internal/core/platform.rs

```diff
@@ -290,6 +290,19 @@ pub enum WindowEvent {
         /// The new logical size of the window
         size: LogicalSize,
     },
+    /// The user requested the window to close
+    ///
+    /// The backend should send this event when the user tries to close the window (for example by pressing the close button)
+    ///
+    /// This will have the effect to send the callback set in [`Window::on_close_requested()`](`crate::api::Window::on_close_requested()`)
+    /// and then hiding the window depending on the return value of the callback
```

Comment:
```suggestion
    /// This will have the effect if invoking the callback set in [`Window::on_close_requested()`](`crate::api::Window::on_close_requested()`)
    /// and then hiding the window depending on the return value of the callback.
```